subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -38,7 +38,7 @@ static enum execution_status execution_state = AWS_JOBS_QUEUED;
 static enum fota_status fota_state = NONE;
 
 /* Document version starts at 1 and is incremented with each accepted update */
```

Comment:
Should probably update  comment that the initial value is parsed out of the job document.